receiver/jmxmetricsreceiver/config.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jmxmetricsreceiver
+
+import (
+	"fmt"
+	"time"
+
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+type config struct {
+	configmodels.ReceiverSettings `mapstructure:",squash"`
+	// The target JMX service url.
+	ServiceURL string `mapstructure:"service_url"`
+	// The script for the metric gatherer to run on the configured interval.
+	GroovyScript string `mapstructure:"groovy_script"`
+	// The endpoint for the metric gatherer to use to report metrics.
+	CollectorEndpoint string `mapstructure:"collector_endpoint"`
```

Comment:
Do we need to support this? Could be confusing to have metrics flow out of band from a receiver. Can we just always feed the metrics through the pipeline and let the user decide how to send them via exporter?

Reply:
It's unclear to be how this is fully able to operate w/ the provided utilities in the extension, but I've removed this option to defer the decision until actual functionality is implemented.  otlp and prometheus receivers can be supported, maybe being configured by the receiver creator?